pkg/server/helpers_windows.go

```diff
@@ -19,10 +19,39 @@ limitations under the License.
 package server
 
 import (
+	runhcsoptions "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/options"
+	criconfig "github.com/containerd/cri/pkg/config"
+	"github.com/pkg/errors"
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
 // initSelinuxOpts is not supported on Windows.
 func initSelinuxOpts(selinuxOpt *runtime.SELinuxOption) (string, string, error) {
 	return "", "", nil
 }
+
+func (c *criService) getSandboxPlatform(sandboxID string) (string, error) {
+	sandbox, err := c.sandboxStore.Get(sandboxID)
+	if err != nil {
+		return "", err
+	}
+	var sandboxPlatform string
```

Comment:
No need for this. Can just do `sandboxPlatform := rhcso.SandboxPlatform` on line 52 and then if the if condition is true it will get reassigned.